### PR TITLE
Manhastro/Madara: Fix nullpointer when manga title is set

### LINKED ISSUE #1416: Manhastro nullpointer error
### Source information

Manhastro

### Source language

Brazilian Portuguese

### Steps to reproduce

1. install Manhastro
2. Select any manga
3. Nullpointer error popup is thrown

### Expected behavior

Should load manga details

### Actual behavior

Throws an error on load manga details

### Mihon/Tachiyomi version

0.15.2.4

### Android version

Android 13

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by choppeh**
Closes #1416

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

**Comment by choppeh** [comment]
I don't know whether to add the bug fix specifically in the extension or add it to the parent class. I did both, but sent a PR with correction in Madara's source

File: lib-multisrc/madara/src/eu/kanade/tachiyomi/multisrc/madara/Madara.kt [hunk 1]
```diff
@@ -723,7 +723,7 @@ abstract class Madara(
     }
 
     // Manga Details Selector
-    open val mangaDetailsSelectorTitle = "div.post-title h3, div.post-title h1, #manga-title > h1"
+    open val mangaDetailsSelectorTitle = "div.post-title h3, div.post-title h1, #manga-title > h1, div.summary_content h2"
```
Comment (AwkwardPeak7) [inline lib-multisrc/madara/src/eu/kanade/tachiyomi/multisrc/madara/Madara.kt:726]: do you know if any other source uses this selector as well? if not then just override the specific source.

Comment (choppeh) [reply lib-multisrc/madara/src/eu/kanade/tachiyomi/multisrc/madara/Madara.kt:726]: Ok

Comment (choppeh) [reply lib-multisrc/madara/src/eu/kanade/tachiyomi/multisrc/madara/Madara.kt:726]: I searched through a few sources, but didn't find another matching manga details title identifier

**Pull request closed without merging**